networks/rpc/server.go

```diff
@@ -350,6 +363,7 @@ func (s *Server) handle(ctx context.Context, codec ServerCodec, req *serverReque
 			notifier, _ := NotifierFromContext(ctx)
 			notifier.activate(subid, req.svcname)
 		}
+		*subCnt++
```

Comment:
Isn't atomic operation needed here?

Reply:
Doesn't a lock needed for `subCnt`? it is accessed from several go routines.

Reply:
I also agree with Melvin's comment.

Reply:
You are right! Thanks! I changed it to atomic operation.